source/_reference-g-s.html.md.erb

```diff
@@ -5307,6 +5312,22 @@ Represents the project settings dependencies for a specific sheet.  Project sett
 **readWriteShowToolbar** | Boolean | **DEPRECATED** Indicates whether the left nav toolbar is displayed. The default, or **true**, is to display the toolbar. If **false**, hides the toolbar.
 **readWriteUrl** | string | URL for 'Edit by Anyone' view of the published sheet<br/>Only returned in a response if **readWriteEnabled = true**.
 
+### SheetSummary Object
+
+Represents the entire summary, or a list of defined fields and values, for a specific sheet.
+
+     |     |     |
+-----|-----|-----|
+**fields** | SummaryField[] | Array of summary (or metadata) fields defined on the sheet.
+
+### SheetUserPermissions Object
+
+Describes the current user's editing permissions for a specific sheet.
+
+     |     |     |
+-----|-----|-----|
+**summaryPermissions** | string | One of <ul><li>**ADMIN**: full control over fields</li><li>**READ_DELETE**: sheet is owned by an individual account that doesn't have summary capabilities. If a summary exists, the only possible operations are GET and DELETE fields.</li><li>**READ_ONLY**</li><li>**READ_WRITE**: can edit values of existing fields, but not create or delete fields, nor modify field type.</li></ul>
```

Comment:
:thinking: "READ_DELETE [...] If a summary exists, the only possible operations are GET and DELETE fields." 

This isn't strictly true. The /summary/fields `PUT` operation can support adding, deleting, and updating fields (we've been calling it MEGA_PUT internally). So technically speaking, I'm pretty sure one could delete fields via `PUT`. I also understand we may not want to publicly document the complex `PUT` function that supports more than a true REST function, and may confuse customers.

Reply:
@kfansler we have not used the updateAction property of the PUT publicly, so my $.02 would be to leave this.

Reply:
Agreed. I think we may have told customers that a workaround is to PUT a different value in some cases, but best to leave out of this formal documentation.